src/inet-commands.c

```diff
@@ -426,11 +426,6 @@ void inet_register(struct state *state, char **fdnames, char **srvnames)
 
 			int r = inet_register_socket(state, fd, name);
 			if (r == -1) {
-				fprintf(stderr,
-					"[!] Socket must have SO_REUSEPORT "
-					"set!\n");
-			}
-			if (r == -2) {
```

Comment:
I _really_ wanted to do the AF_UNSPEC trick here to reopen socket with REUSEPORT....

Reply:
That explains. Racy, however, because someone could steal the port from under us.

Anyway, not needed now that we switched to SOCKMAP.